notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-756_analysis.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # await client.get_fields("lsst.sal.MTM1M3.hardpointActuatorData")
</pre><p>Same as above. Is it worth keeping this commented cell ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>